ResearchSuite/ResearchSuite/RSDResourceTransformer.swift

```diff
@@ -95,19 +95,20 @@ public enum RSDResourceTransformerError : Error, CustomNSError {
 }
 
 public protocol RSDResourceTransformer {
-    var classType: String? { get }
-    var resourceName: String? { get }
+    var resourceName: String { get }
     var resourceBundle: String? { get }
+    var classType: String? { get }
 }
 
 extension RSDResourceTransformer {
     
+    public var estimatedFetchTime: TimeInterval {
+        return 0
```

Comment:
I thought things defined in protocol extensions couldn't be overridden in implementations...?